### DISCUSSION TITLE
can not make call

### DISCUSSION THREAD

**Discussion by wjswlgus123** [IchthysMaranatha/asterisk-chan-quectel#3] [Q&A]
-- Called Quectel/quectel0/010xxxxx
    -- Quectel/quectel0-010000000c is making progress passing it to 
       > 0x7feec0031cd0 -- Strict RTP switching to RTP target address as source
[Nov 26 14:39:16] WARNING[3257][C-0000000e]: app_dial.c:1680 wait_for_answer: Unable to write frametype: 2
[Nov 26 14:39:16] WARNING[3257][C-0000000e]: app_dial.c:1680 wait_for_answer: Unable to write frametype: 2
[Nov 26 14:39:16] WARNING[3257][C-0000000e]: app_dial.c:1680 wait_for_answer: Unable to write frametype: 2
[Nov 26 14:39:16] WARNING[3257][C-0000000e]: app_dial.c:1680 wait_for_answer: Unable to write frametype: 2
[Nov 26 14:39:16] WARNING[3257][C-0000000e]: app_dial.c:1680 wait_for_answer: Unable to write frametype: 2
[Nov 26 14:39:16] WARNING[3257][C-0000000e]: app_dial.c:1680 wait_for_answer: Unable to write frametype: 2
[Nov 26 14:39:16] WARNING[3257][C-0000000e]: app_dial.c:1680 wait_for_answer: Unable to write frametype: 2
[Nov 26 14:39:16] WARNING[3257][C-0000000e]: app_dial.c:1680 wait_for_answer: Unable to write frametype: 2
[Nov 26 14:39:16] WARNING[3129]: at_response.c:2138 at_response: [quectel0] Receive NO CARRIER

**Comment by IchthysMaranatha** [comment]
Please post minicom result here, we'll carry on the discussion here

**Comment by IchthysMaranatha** [comment]
Entire call process should be working with minicom, except voice. ATD, ATA, ATH, connecting call with correct dsci, all of these should be working properly.
